addon/-private/system/relationships/ext.js

```diff
@@ -171,7 +171,7 @@ export const RelationshipsClassMethodsMixin = Ember.Mixin.create({
     });
    ```
 
-    Calling `App.Post.typeForRelationship('comments')` will return `App.Comment`.
+    Calling `postRecord.constructor.typeForRelationship('comments')` will return `Comment`.
```

Comment:
If I recall correctly, @fivetanley had some thoughts of not advertising `constructor` publicly? Just want to check...